cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -543,3 +544,9 @@ defaultUtxoKeys n =
 plutusV3Script :: Text
 plutusV3Script =
   "{ \"type\": \"PlutusScriptV3\", \"description\": \"\", \"cborHex\": \"46450101002499\" }"
+
```

Comment:
A comment explaining how this was generated here, would be great.

Reply:
I want to add this process to the documentation and then link to it. @CarlosLopezDeLara where should this documentation go?